quickstack/manifests/controller_common.pp

```diff
@@ -866,6 +866,8 @@
     keep_days      => $backups_keep_days,
   }
      
+  class { 'ceilometer::client::controller': }
+
```

Comment:
Add a flag to enable/disable ceilometer on controller and compute nodes from hiera file.

Lets say there is some variable "enable_ceilometer" and if it is set to false in hiera file, ceilometer won't be installed. If it is set to true, only then it will be installed. This will make sure that if someone is using staging areas, we can simply disable ceilometer for them.

Also, in the message of pull request, put the variables which needs to be added in hiera file. Reference pull request for the same:-
https://github.com/CCI-MOC/kilo-puppet/pull/42